index.html

```diff
@@ -3339,6 +3339,23 @@ <h1>DID Methods</h1>
             <a href="https://github.com/identify202020/did-method/blob/main/did_kr_method.md">Korea Mobile Identity System DID Method</a>
           </td>
         </tr>
+        <tr>
+          <td>
+            did:unisot:
+          </td>
+          <td>
+            PROVISIONAL
+          </td>
+          <td>
+            Bitcoin SV
+          </td>
+          <td>
+            <a href="https://www.unisot.com">UNISOT AS</a>
+          </td>
+          <td>
+            <a href="https://gitlab.com/unisot-did/unisot-did-method-specification">UNISOT DID Method</a>
```

Comment:
Non blocking feedback:

```
"publicKey": [{
    "id": "did:unisot:1CFPke8f4Bs611FnTASrVvJsugSbZsvqdz#key1",
    "type": "secp256k1",
    "controller": "did:unisot:1NF4xwFj8tJEv3XGjMNzfktaRJVFbCVBLX",
    "publicKeyBase58": "KmoEcdMASm6A3eNPdLkwQmwtKo...CFG6LCqJN2UC8avj2TwCR6S"
  }],
```

type should be https://w3c.github.io/did-spec-registries/#ecdsasecp256k1verificationkey2019 if you would like interoperability... you may also wish to use `publicKeyJwk` which supports secp256k1.

You may want to update:

```
`"@context": "https://w3id.org/did/v1",`
```

to https://www.w3.org/ns/did/v1

otherwise looks like it meets the requirements.

Reply:
Thanks for noticing that key type error, it was a leftover from an earlier draft.
I also updated the context URL.

Mirko